modules/azure/virtual-machine/main.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  dd_tags = {
+    Datadog                 = "true"
+    DatadogAgentlessScanner = "true"
+  }
+}
+
+resource "azurerm_orchestrated_virtual_machine_scale_set" "vmss" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  tags                = merge(var.tags, local.dd_tags)
+
+  sku_name                    = var.instance_size
+  instances                   = var.instance_count
+  platform_fault_domain_count = 1
+
+  identity {
+    type = "UserAssigned"
+    identity_ids = [
+      var.user_assigned_identity
+    ]
+  }
+  os_profile {
+    linux_configuration {
+      computer_name_prefix            = "dd-agentless-"
```

Comment:
We prefix them with `agentless-scanning-*` on AWS. Would it make sense to be consistent here ?

Reply:
I think I ran into a string length limitation, but I can double-check.

Reply:
Done.